src/Features/Core/Portable/CodeFixes/FixAllOccurrences/FixSomeCodeAction.cs

```diff
@@ -38,26 +36,24 @@ protected override async Task<IEnumerable<CodeActionOperation>> ComputeOperation
             cancellationToken.ThrowIfCancellationRequested();
             FixAllLogger.LogState(FixAllState, IsInternalCodeFixProvider(FixAllState.CodeFixProvider));
 
-            var service = FixAllState.Project.Solution.Workspace.Services.GetService<IFixAllGetFixesService>();
+            var service = FixAllState.Project.Solution.Workspace.Services.GetRequiredService<IFixAllCodeFixGetFixesService>();
 
             var fixAllContext = new FixAllContext(FixAllState, progressTracker, cancellationToken);
-            if (progressTracker != null)
-                progressTracker.Description = FixAllContextHelper.GetDefaultFixAllTitle(fixAllContext);
+            progressTracker.Description = FixAllContextHelper.GetDefaultFixAllTitle(fixAllContext);
```

Comment:
do we know we always have a progress tracker?  is the method annotated properly? are all callers NRT safe?

Reply:
Yes, I explicitly checked all the callers. ProgressTracker passed along in the FixAll system is non-null in all the code paths.